### ISSUE TITLE
Gradients not found in maml? why?

### ISSUE THREAD

**Issue by brando90** [learnables/learn2learn#335]
I got this bizarre error:
```
  File "/home/miranda9/miniconda3/envs/metalearning_gpu/lib/python3.9/site-packages/learn2learn/algorithms/maml.py", line 169, in adapt
    self.module = maml_update(self.module, self.lr, gradients)
UnboundLocalError: local variable 'gradients' referenced before assignment
```

seems weird since it's coming from l2l. Any ideas?

**Comment by Chhinna** [comment]
yeah, getting the same error

**Comment by seba-1511** [comment]
Closing: fixed in #387.
